docs/concepts/getting-started.md

```diff
@@ -41,4 +41,4 @@ The Flow blockchain implements core functionality using its own smart contract l
 
 ## FLOW Token
 
-The [FLOW](./core-contracts/03-flow-token.md) token is the native currency for the Flow network. Developers and users can use FLOW to transact on the network. Developers can integrate FLOW directly into their apps for peer-to-peer payments, service charges, or consumer rewards. Flow can be held, transferred, or transacted peer-to-peer.
+The [FLOW](./core-contracts/03-flow-token.md) token is the native currency for the Flow network. Developers and users can use FLOW to transact on the network. Developers can integrate FLOW directly into their apps for peer-to-peer payments, service charges, or consumer rewards. FLOW can be held, transferred, or transacted peer-to-peer.
```

Comment:
Probably a typo

Reply:
market names of tokens are usually in capital case, like BTC, ETH, SHIB, MATIC:
https://www.coinbase.com/price/flow